src/Script/Utility/Fragment.js

```diff
@@ -0,0 +1,24 @@
+export default class Fragment
+{
+    static createFragment(urlFragment)
+    {
+        window.location.hash = `#${urlFragment}`;
+    }
+
+    static readFragment()
+    {
+        if (window.location.hash == '') { return('No Url Fragment loaded')}
```

Comment:
No need for the no url fragment loaded message, it will return empty automatically